.github/scripts/pr-backport.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+TARGET_REMOTE=upstream
+KEYCLOAK_REPO=https://github.com/keycloak/keycloak
+
+PR=$1
+TARGET=$2
+
+function echo_header() {
+  echo ""
+  echo "======================================================================="
+  echo "$1"
+  echo "-----------------------------------------------------------------------"
+}
+
+function error() {
+  echo "======================================================================="
+  echo "Error"
+  echo "-----------------------------------------------------------------------"
+  echo "$1"
+  echo ""
+  exit 1
+}
+
+if ! [ -x "$(command -v gh)" ]; then
+  error "The GitHub CLI is not installed. See: https://github.com/cli/cli#installation"
+fi
+
```

Comment:
Perhaps we can check that jq is installed as well.
```suggestion
if ! [ -x "$(command -v jq)" ]; then
  error "The jq command is not installed. Use: sudo dnf install jq or equivalent on your system."
fi
```

Reply:
We can link to the docs for installing JQ here: https://jqlang.github.io/jq/download/

Reply:
Done.